packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-classes--to-es2015/replacement-static-decorator-initializer-this/exec.js

```diff
@@ -0,0 +1,39 @@
+
+let original, replaced, accessorThis, getterThis, setterThis, methodThis, propertyThis;
+
+function dec(Klass) {
+  original = Klass;
+  replaced = class extends Klass {};
+
+  return replaced;
+}
+
+function captureInitializerThis(callback) {
+  return function (_, context) {
+    context.addInitializer(function () {
+      callback(this);
+    })
+  }
+}
+
+@dec
+class Foo {
+  @(captureInitializerThis(v => accessorThis = v))
+  static accessor;
```

Comment:
Oops, merged too early, it should have been `static accessor accessor`.

Reply:
I committed this change at https://github.com/babel/babel/pull/16287. :)